spec/unit/facter/sudo_version_spec.rb

```diff
@@ -1,57 +1,61 @@
 require 'spec_helper'
 require 'facter/sudo_version'
 
-describe Facter::Util::SudoVersion do
+describe 'sudo_version facts' do
+  subject(:fact) { Facter.fact(:sudo_version) }
+
+  before :each do
+    # Ensure we're populating Facter's internal collection with our Fact
+    SudoVersion.add_facts
+  end
 
   with_fixtures = {
     'Debian' => {
-      :sudo_output => 'Sudo version 1.8.3p1\nSudoers policy plugin version 1.8.3p1\nSudoers file grammar version 40\nSudoers I/O plugin version 1.8.3p1\n',
```

Comment:
True, but we wouldn't know without the whole output. What if we want to tweak our regex? When it comes to fixtures, I think we should take the entire output.